darwin/DarwinMachine.c

```diff
@@ -67,6 +67,16 @@ static void DarwinMachine_getVMStats(vm_statistics_t p) {
    }
 }
 
+#if defined(__arm64__)
+static void DarwinMachine_getVMStats64(vm_statistics64_t p) {
```

Comment:
Consider merging this function with `DarwinMachine_getVMStats` above, so as to avoid duplicate code.

You can use "`DarwinMachine* this`" as the input argument.